nss-day-cohort-56/rare-client-monstrous_mambas
# My Posts and Delete Button

# Description
personal posts are viewable under My Posts. you are able to delete these posts as well

Please delete options that are not relevant.
- [x ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- please create a post through sql or postman using your person user_id
- log into client side with your username and make sure that post is in My Posts
- delete that post using the Delete button

# Checklist:
- [ x] I have performed a self-review of my own code
